commands/command.cpp

```diff
@@ -1,10 +1,10 @@
 // SPDX-License-Identifier: GPL-2.0
 
 #include "command.h"
-#include "command_divelist.h"
-#include "command_divesite.h"
-#include "command_edit.h"
-#include "command_edit_trip.h"
+#include "divelist.h"
+#include "divesite.h"
+#include "edit.h"
+#include "edit_trip.h"
 
```

Comment:
I don't think I'd do this - now we have two `divelist.h` files in `core` and in `commands`. That's just asking for trouble.

Reply:
What do you suggest? `commands/command_divelist.h`?

Note that the only header included from outside is `commands/command.h`. Everything else is purely local to the commands-folder.

Reply:
Yes, I think `#include "commands/command_divelist.h"` makes sense.
Admittedly, a bit verbose, but certain to avoid confusion :-)

Reply:
Fine with me.